src/files/6_StyleCopAnalyzers.editorconfig

```diff
@@ -0,0 +1,988 @@
+# global_level must be higher than the NET Analyzer files
```

Comment:
I used reflection to extract the list of all the StyleCop rules to disable them by default, except one: SA1649